### ISSUE TITLE
Release 1.4.4

### ISSUE THREAD

**Issue by birchamp** [unfoldingWord/tc-create-app#1290]
- [x] bump the version the PRs 
- [x] tag the version when merged. 

#### Prepare Staging 
- [x] PR to remove RC from the version number
  - start in new branch (e.g., `zpp-remove-rc-1.4.4`); 
  - edit `package.json` for version (e.g., v1.4.4); commit; publish
  - `git tag` (e.g., v1.4.4);    `git push --tags`
  - create PR from `remove...` branch --> `develop`;  connect this PR to the "RELEASE" issue
  - merge this PR (even if you merge it yourself, please use PR and do not commit directly to the develop branch)
- [x]  Release Branch:
  - pull latest develop changes
  - create new branch: release branch for version 1.4.4 from develop
  - publish release branch to github
- [x] Create staging Netlify link (PROD Data)
  - create new PR from `v1.4.4` --> `STAGE`; connect to the release issue
  - walk through happy path on deploy preview
  - merge 

Definition of Done
- [x]  QA signs off on specific build in regression testing: current for v4.1.0, build number: 
- [x]  Update changelog
- [x]  create draft release on github for translationCore
- [x]  Attach the github action builds (same build as QA signed off) to the draft release as assets
- [x]  Update Release Notes to the draft release
 after everyone has reviewed translationCore draft release, make it an official release.
- [x]  Confirm all links to the release
-  github release page - https://github.com/unfoldingWord/translationCore/releases
-  door43 forum

**Comment by BincyJ** [comment]
@Joel-C-Johnson , @jincypjose

**Comment by BincyJ** [comment]
@Joel-C-Johnson , @jincypjose , @elsylambert ,

here are the release notes :

What's New? 
We are pleased to announce that tC Create 1.4.4 is now available online at https://create.translationcore.com.

-Editing in the preview mode created many formatting issues in the text. We have disabled editing within preview mode. Users can now only view text in preview mode while they can continue to edit in the default mark-down mode.
[View this issue on GitHub](https://github.com/unfoldingWord/tc-create-app/issues/1127)

Users had a difficult time clicking the editable textbox as the cursor navigation experience was not friendly. We have now fixed it so that the clicking anywhere within the outline of the text box should allow the user to begin editing.
[View this issue on GitHub]https://github.com/unfoldingWord/tc-create-app/issues/1153()
